Tukui/Libs/oUF_AuraTrack/oUF_AuraTrack.lua

```diff
@@ -369,7 +369,7 @@ local UpdateIcon = function(self, unit, spellID, texture, id, expiration, durati
 	end
 
 	local PositionX = (id * AuraTrack.IconSize) - (AuraTrack.IconSize) + (AuraTrack.Spacing * id)
-	local r, g, b = unpack(Tracker[spellID])
+	local r, g, b = unpack(AuraTrack.Tracker[spellID])
```

Comment:
```suggestion
	local r, g, b = unpack(Tracker[spellID])
```
The var Tracker is declared at line 3 and is set based if isRetail or not on line 24 or line 81 depending if true or not.  With ur isRetail changes, you should revert this part of the commit and just test with ur isRetail changes.

Reply:
I changed to `AuraTrack.Tracker`  because if you overwrite the `Tracker` spell table inside the Raid.lua file, since you are using the local Tracker variable the `unpack` function throws a error, if the  spellID do not exists there.

This happens because at line 489 where is this validation `if (self.AuraTrack.Tracker[spellID]) and (caster == "player" or caster == "pet") then` using `self.AuraTrack.Tracker`.

Reply:
ahh i see what you mean.  you are prob right after looking through it. just something that seemed off when scanning through ur pr and it didnt "seem" right at first. I personally didnt have time to test myself which is why i suggested u to verify my thoughts, in which thank you for providing the feedback and let me know.  Give tukz some time as he is busy irl to verify ur changes and do the pr.